pyaedt/modeler/GeometryOperators.py

```diff
@@ -1448,3 +1462,354 @@ def orient_polygon(x, y, clockwise=True):
+            If ``False`` the left-handed rotation from Va to Vb is considered. The default is ``True`.
+
+        Returns
+        -------
+        float
+            Angle in radians.
+
+        """
+        c = va[0] * vb[1] - va[1] * vb[0]
+
+        if righthanded:
+            return math.atan2(c, GeometryOperators.v_dot(va, vb))
+        else:
+            return math.atan2(-c, GeometryOperators.v_dot(va, vb))
+
+    @staticmethod
+    @aedt_exception_handler
+    def point_in_polygon(point, polygon):
+        """
+        Determine if a point is inside or outside a polyogn, both located on the same plane.
+        The method implements the radial algorithm (https://es.wikipedia.org/wiki/Algoritmo_radial)
+
+        point : list
+            List of ``[x, y]`` coordinates.
+        polygon : list
+            [[x1, x2, ..., xn],[y1, y2, ..., yn]]
+
+        Returns
+        -------
+        int
+            -1 When the point is outside the polygon.
+            0 When the point is exactly on one of the sides of the polygon.
+            1 When the point is inside the polygon.
+        """
+        # fmt: off
+        tol = 1e-8
+        assert len(point) == 2, "point must be a list in the form [x, y]"  # pragma: no cover
+        pl = len(polygon[0])
+        assert len(polygon[1]) == pl, "Polygon x and y lists must be the same length"  # pragma: no cover
+        asum = 0
+        for i in range(pl):
+            vj = [polygon[0][i-1], polygon[1][i-1]]
+            vi = [polygon[0][i], polygon[1][i]]
+            if GeometryOperators.points_distance(point, vi) < tol:
+                return 0  # point is one of polyline vertices
+            vpj = GeometryOperators.v_points(point, vj)
+            vpi = GeometryOperators.v_points(point, vi)
+            a = GeometryOperators.v_angle_sign_2D(vpj, vpi)
+            if abs(abs(a) - math.pi) < tol:
+                return 0
+            asum += a
+        if abs(asum) < tol:
+            return -1
+        elif abs(asum - 2*math.pi) < tol:
+            return 1
+        else:  # pragma: no cover
+            raise Exception("Unexpected error!")
+        # fmt: on
+
+    @staticmethod
+    @aedt_exception_handler
+    def is_point_in_polygon(point, polygon):
+        """
+        Determine if a point is inside or outside a polyogn, both located on the same plane.
+        The method implements the radial algorithm (https://es.wikipedia.org/wiki/Algoritmo_radial)
+
+        point : list
+            List of ``[x, y]`` coordinates.
+        polygon : list
+            [[x1, x2, ..., xn],[y1, y2, ..., yn]]
+
+        Returns
+        -------
+        bool
+            ``True`` if the point is inside the polygon or xactly on one of its sides.
+            ``False`` otherwise.
+        """
+        r = GeometryOperators.point_in_polygon(point, polygon)
+        if r == -1:
+            return False
+        else:
+            return True
```

Comment:
Can be simplified with:
```suggestion
        return r == -1
```

Reply:
Yes, I know and usually I'd do what you suggested. 
Actually, in this case, it would be ``return r != -1`` or ``return not r== -1``. So in this case I've chosen readability over verbosity.